src/lib/progress.h

```diff
@@ -95,6 +95,8 @@ struct step progress_get_step(void);
  */
 void progress_complete_step(void);
 
+void progress_report_interdeterminate(void);
```

Comment:
this declaration is missing its explanation

Reply:
Done